packages/console/src/containers/AppEndpointsProvider/index.tsx

```diff
@@ -0,0 +1,40 @@
+import ky from 'ky';
+import type { ReactNode } from 'react';
+import { useMemo, useEffect, createContext, useState } from 'react';
+
+import { adminTenantEndpoint, userTenantId } from '@/consts';
+
+type Props = {
+  children: ReactNode;
+};
+
+export type AppEndpoints = {
+  userEndpoint?: URL;
+  adminEndpoint?: URL;
+};
+
+export type AppEndpointKey = keyof AppEndpoints;
+
+export const AppEndpointsContext = createContext<AppEndpoints>({});
+
+const AppEndpointsProvider = ({ children }: Props) => {
```

Comment:
Can we use `useSWR` instead of custom context?

Reply:
`useSWR` will automatically reload i think - as this should be treated as "fixed" once the console is loaded